pymeasure/instruments/kuhneelectronic/kusg245_250a.py

```diff
@@ -0,0 +1,414 @@
+        b = self.read_bytes(2)
+        if _has_correct_termination_character(b):
+            return int.from_bytes(b[:1]) / 256.0 * 360.0
+        raise ConnectionError(_err_msg_invalid_termination_character(b))
+
+    @phase_shift.setter
+    def phase_shift(self, value):
+        value = int(round(truncated_range(value, [0, 358.6])) / 360.0 * 256.0)
+        self.write(f"H{value:03d}")
+
+    @property
+    def reflection_limit(self):
+        "B%d",
+        """Limit of reflection in Watts (integer in 0 - no limit, 100, 150, 180, 200, 230).
+
+        .. note::
+
+            If the limit for the reflected power is reached, the forward power
+            is reduced to the specified value and the power control mechanism
+            is locked until the alarm has been cleared by the user via
+            :meth:`~.Kusg245_250A.clear_VSWR_error()`.
+        """
+        self.write("B?")
+        b = self.read_bytes(2)
+        if _has_correct_termination_character(b):
+            return reflection_limit_map[b[0]]
+        raise ConnectionError(_err_msg_invalid_termination_character(b))
+
+    @reflection_limit.setter
+    def reflection_limit(self, value):
+        value = truncated_discrete_set(value, reflection_limit_map.values())
+        inv_reflection_limit_map = {v: k for k, v in reflection_limit_map.items()}
+        value = inv_reflection_limit_map[value]
+        self.write(f"B{value:d}")
+
+    def tune(self, power):
+        """
+        Find and set the frequency with lowest reflection
+        at a given power.
+
+        :param power: A power set-point for tuning (in Watts).
+                      (integer from 0 to :attr:`power_limit`
+                      parameter - see constructor).
+        """
+        power = truncated_range(power, [0, self._power_limit])
+        self.write(f"b{power:03d}")
+
+    def clear_VSWR_error(self):
+        """
+        Clears the VSWR error.
+
+        See: :attr:`~.Kusg245_250A.reflection_limit`.
+        """
+        self.write("z")
+
+    def store_settings(self):
+        """
+        Save actual settings to EEPROM.
+
+        The following parameters are saved:
+        frequency mode (see :attr:`~.Kusg245_250A.freq_steps_fine_enabled`),
+        frequency (see :attr:`~.Kusg245_250A.frequency_coarse`
+        or :attr:`~.Kusg245_250A.frequency_fine`),
+        output power set-point (see :attr:`~.Kusg245_250A.power_setpoint`),
+        ON/OFF control setting (see :attr:`~.Kusg245_250A.external_enabled`),
+        reflection limit (see :attr:`~.Kusg245_250A.reflection_limit`),
+        on time for pulse mode (see :attr:`~.Kusg245_250A.pulse_width`) and
+        off time for pulse mode (see :attr:`~.Kusg245_250A.off_time`).
+        """
+        self.write("SE")
+
+    def shutdown(self):
+        """
+        Safe shut-down the generator.
+
+        1. Disable RF output.
+        2. Deactivate biasing.
+        """
+        self.rf_enabled = False
+        self.bias_enabled = False
+
```

Comment:
Please add a super call, such that the parents shutdown is called:
```suggestion
        super().shutdown()

```